drivers/ethernet/eth_stm32_hal.c

```diff
@@ -211,8 +217,9 @@ static HAL_StatusTypeDef read_eth_phy_register(ETH_HandleTypeDef *heth,
 						uint32_t PHYReg,
 						uint32_t *RegVal)
 {
-#if defined(CONFIG_SOC_SERIES_STM32H7X) ||  defined(CONFIG_SOC_SERIES_STM32H5X) || \
-	defined(CONFIG_ETH_STM32_HAL_API_V2)
+#if defined(CONFIG_SOC_SERIES_STM32H7X) || defined(CONFIG_SOC_SERIES_STM32H5X)
```

Comment:
if MDIO ?

Reply:
Here is the same, still not implemented and tested MDIO support for F7 serie at this stage.

Reply:
Since MDIO is only available on H5 and H7 today, then `if MDIO` is the same as `if H5 || H7`.
Benefit is that you won't need to do any modification when MDIO will support F7.

Reply:
@spectrum70 Can you check on my feedback on this point ?

Reply:
Ok, that's correct. Let me change and test it.

Reply:
@erwango  fixed, thanks

Reply:
yes, correct, select is what i want.
Fixed, thanks.